lib/frizz/remote.rb

```diff
@@ -39,7 +50,9 @@ def delete(remote_file)
     attr_reader :bucket_name, :ignorance, :options
 
     def ignore?(object)
-      ignorance.ignore?(object.key)
+      # for remote files, you don't need to ignore for gzip preference,
+      # since there won't be any gzip files on s3
+      ignorance.ignore_matched_pattern?(object.key)
```

Comment:
I don't like how we've leaked the implementation of ignorance into both `Remote` and `Local` ... is there a way we can keep the matched_pattern vs gzip stuff internal to the implementation of `Ignorance#ignore?` _and_ address the local vs full path issue?